commercetools/machine-learning/commercetools-ml-api/src/main/java/com/commercetools/api/ml/generated/client/ByProjectKeyMissingDataAttributesPost.java

```diff
@@ -6,6 +6,8 @@
 import java.io.InputStream;
 import java.io.IOException;
 
+import java.nio.file.Files;
```

Comment:
And we now in introduced this unnecessary import in all files and that's why this PR is so big... I see 😉

Reply:
Yes, i mean the way to avoid that is to use a post generation optimizer, other than that the logic 🤷